test/test-mongocrypt-ctx-encrypt.c

```diff
@@ -2551,8 +2606,24 @@ _test_encrypt_fle2_unindexed_encrypted_payload (_mongocrypt_tester_t *tester)
    uint8_t rng_data[] = RNG_DATA;
    _test_rng_data_source source = {
       .buf = {.data = rng_data, .len = sizeof (rng_data) - 1u}};
-   _test_encrypt_fle2_encryption_placeholder (
-      tester, "fle2-insert-unindexed", &source);
+
+   // TODO (SERVER-74139): Implement InsertUpdatePlacePayloadV2 transform
```

Comment:
```suggestion
   // TODO (MONGOCRYPT-551) Implement UnindexedEncryptedValueV2 transform
```

Reply:
Hah! I knew there had to be a ticket for this. Thanks for finding it!